Add funding section (#469)
astropy/astropy-project: roadmap/roadmap.md

- :large_orange_diamond: Document the process for taking on a formal role in the Astropy project and recruit for unfilled team positions.

- :large_orange_diamond: Define and staff the  strategic planning role.

## Fund Raising

> The broad goal in this section is to ensure Astropy has funding to support development that volunteers cannot or will not do on their own funding. To ensure project sustainability, this needs to be funding sources that can be maintained over the long term.

Reviewer: ```suggestion
To ensure project sustainability, the Astropy Project needs funding sources that can be maintained over the long term.
```

Reviewer 2: Nitpick: Break into shorter lines.

- :red_square: Raise funds from governmental sources outside of the US

> - :large_orange_diamond: Raise funds from private foundations to give us independence from political vagaries.

Reviewer 2: Just throwing out some alternative verbiage. Not sure if better or worse.
```suggestion
- :large_orange_diamond: Raise funds from private foundations to give us independence from geopolitics.
```

> - :red_square: Raise funds from governmental sources outside of the US
> - :large_orange_diamond: Raise funds from private foundations to give us independence from political vagaries.
> - :green_circle: Maintain existing funding sources (e.g., NASA)

Reviewer: I would suggest keeping the roadmap very neutral.
```suggestion
- :red_square: Raise funds from governmental sources outside of the US.
- :large_orange_diamond: Raise funds from private foundations.
- :green_circle: Maintain existing funding sources (e.g., NASA).
```
